### DISCUSSION TITLE
Not able to use LiveActivityEvent class in the depenedcy

### DISCUSSION THREAD

**Discussion by vincedprime** [relayrides/pushy#1055] [Q&A] [answered]
I am using 0.15.5-SNAPSHOT.
Not sure why I'm not able to get LiveActivityEvent enum. Can use other utility class
<img width="575" alt="image" src="https://github.com/jchambers/pushy/assets/62996220/91c42c7e-ad2b-4d03-aaa9-a0e18cf42c2e">
<img width="300" alt="image" src="https://github.com/jchambers/pushy/assets/62996220/dc4e6e4e-65fb-4660-9ead-e680546f0125">

**Comment by vincedprime** [comment]
It's working with 0.15.2. But only the UPDATE and END activity are present
<img width="298" alt="image" src="https://github.com/jchambers/pushy/assets/62996220/720fe8b3-46ca-469d-b937-1849b9ca9c90">
@jchambers any thing I'm doing wrong?

**Comment by jchambers** [comment]
There is no 0.15.5-SNAPSHOT release. I don't know where you found that. Please try with Pushy 0.15.4, which is the latest actual release.

**Comment by vincedprime** [comment]
Missed context from my end. I was using the latest 0.15.4. Was facing the same issue that's why I checked the [pom file ](https://github.com/jchambers/pushy/blob/main/pom.xml)for the latest version and I didn't cross checked if this was released or not. 

Same issue with 0.15.4
<img width="285" alt="image" src="https://github.com/jchambers/pushy/assets/62996220/1ef9da0f-728a-4131-a9c3-eddb16f35665">

**Reply by jchambers** [reply, marked as answer]
In short: I don't know. This sounds like a problem with your local development environment. I am unable to reproduce this problem—or what I understand the problem to be from your screenshots—locally.

Can you please clarify what the actual error message is? Screenshots showing something highlighted in red can give hints, but generally lack the detail needed to really understand a problem.

**Comment by vincedprime** [comment]
Yes, this was only on local env. Dunno something is getting mixed up locally and the application giving error

java: cannot find symbol
  symbol:   variable UPDATE
  location: class com.eatthepath.pushy.apns.util.LiveActivityEvent

But after you mentioned it's something with local, I tried it in our testing env. Worked like a charm.
Sorry for the noise.
